src/org/scada_lts/dao/migration/CorrectMigration2_2.java

```diff
@@ -0,0 +1,66 @@
+package org.scada_lts.dao.migration;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.scada_lts.dao.DAO;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+/**
+ * @autor grzegorz.bylica@gmail.com on 14.09.2020
+ */
+public class CorrectMigration2_2 {
+
+    private static final Log LOG = LogFactory.getLog(CorrectMigration2_2.class);
+
+    public static void correct() {
```

Comment:
Is it a good idea to use drops in migration, why did you approach it this way? It was possible to roll back the changes on the test base by executing queries and purely uploading new changes.